### ISSUE TITLE
always print variable even with --quiet option

### ISSUE THREAD

**Issue by psychobolt** [dotenvx/dotenvx#158]
Hi I noticed, another case when my CI Provider has DOTENV_KEY globally defined, each of our `dotenvx get` commands, will produce the warning and following prints our variable:

```sh
# e.g. try: DOTENV_KEY=foo dotenvx --log-level=info get TARGET_URL
# when there is no vault file
[dotenv@16.4.5][WARN] You set DOTENV_KEY but you are missing a .env.vault file at null. Did you forget to build it?
# variable
```
In my monorepo setup, this would happen for only workspaces without a vault file, but I don't think the `get` command should load vault? I tried to set the quiet command to ignore the warning however, it turns out --quiet also turns off the variable output as well. None of the other `log-level` flags seem to output only the variable.

**Comment by motdotla** [comment]
this is a bug. thank you for pointing it out. working on the fix.

**Comment by motdotla** [comment]
you're our best bug catcher 🥅! effort started here: https://github.com/dotenvx/dotenvx/pull/161

`get` should make re-use of `run`, just not injecting to process.env. that will solve this problem. right now `get` could minor discrepancies with `run` - for example, evaluating commands. it's only supporting expansion currently.

**Comment by psychobolt** [comment]
That makes a lot of sense. I agree. I even made changes to utilize `run` instead for my [scripts](https://github.com/psychobolt/vite-storybook-boilerplate/blob/610d8d2f2d96994daa0efb40f4d41154f6d31c9d/package.json#L9).

**Comment by motdotla** [comment]
all set. released as `v0.30.0`. it uses `run` under the hood and outputs even when `--quiet` flag is passed.
